### PR TITLE
tidying up

### PR DESCRIPTION
**Pull request by halimahexe**
tidied up `index.js` and added `index-test.js` and `ask.js` where I was playing around with slash commands but I haven't figured them out#

also added the conversation history code so the bot can refer back to conversation history

### CODE REVIEW

File: index-test.js
Comment (Leda909) [inline index-test.js:1]: Could you please explain this?
image

Comment (Leda909) [reply index-test.js:1]: Could you explain this as well?
image

Comment (halimahexe) [reply index-test.js:1]: The `index-test.js` is just me playing around with things. I made proper changes to `index.js`.

The first image is using the code from Discord to add created slash commands to a collection as I was wondering whether it's possible to create an `/ask` command that fetches from OpenAI. But I haven't figured that out yet, which is why it's in the test file (so I can test without breaking anything in `index.js`)

And the second is hard-coding a command prefix `/ask` so every message the bot responds to has to start with that.

Comment (Leda909) [reply index-test.js:1]: I see, here you create the array to the userHistory. Which point of the user story ask you to do user story?
image

**Review by Leda909 — APPROVED** [review]

**Pull request merged**